BookReader/BookReader.js

```diff
@@ -1240,12 +1246,23 @@ BookReader.prototype.zoom1up = function(direction) {
 /**
  * Resizes the inner container to fit within the visible space to prevent
  * the top toolbar and bottom navbar from clipping the visible book
+ *
+ * @param { boolean } fillMainContainer - optional
+ * When used, BookReader will fill the main container with the book's content.
+ * This is primarily for 1up view - a follow up animation to the nav animation
+ * So resize isn't perceived sharp/jerky
  */
-BookReader.prototype.resizeBRcontainer = function() {
-  this.refs.$brContainer.css({
-    top: this.getToolBarHeight(),
-    bottom: this.getNavHeight(),
-  });
+BookReader.prototype.resizeBRcontainer = function(fillMainContainer) {
+    if (!fillMainContainer) {
+        return this.refs.$brContainer.css({
+            top: this.getToolBarHeight(),
+            bottom: this.getNavHeight()
+          });
+    }
```

Comment:
Not a blocker, but since these two are mutually exclusive options (and you're not actually returning anything), I might do `if / else` instead of `if / return`.

Reply:
✅